http/obmc_shell.hpp

```diff
@@ -58,45 +58,19 @@ class Handler : public std::enable_shared_from_this<Handler>
         {
             // CHILD
 
-            // sets the effective user ID
-            // as service user
-            bool isUidSet = false;
             if (auto userName = session->getUserName(); !userName.empty())
             {
-                if (struct passwd* pw = getpwnam(userName.c_str());
-                    pw != nullptr)
-                {
-                    int uidr = setuid(pw->pw_uid);
-                    if (!uidr)
-                    {
-                        isUidSet = true;
-                    }
-                    else
-                    {
-                        BMCWEB_LOG_ERROR << "sets service user failed Error: "
-                                         << uidr;
-                    }
-                }
-                else
-                {
-                    BMCWEB_LOG_ERROR << "getpwnam return null passwd";
-                }
+                execl("/bin/login", "/bin/login", "-f", userName.c_str(), NULL);
+                // execl only returns on fail
+                BMCWEB_LOG_ERROR << "unable to replaces current process, execl "
+                                    "fail, Erro number:"
```

Comment:
Nit: Typo: Erro -> Error (missing "r").
If you fix this, please also  change the comment 13 lines above "Error creating ssh child process" because it is not accurate; try: "Error creating child process for login shell."

Reply:
Done